tools/vulnerability-scan/wss-scan.sh

```diff
@@ -23,7 +23,12 @@ set -e
 
 # Generate temporary `settings.gradle` file based on the name in `build.gradle`
 function generate_settings_gradle() {
-  settings_gradle_content=`cat build.gradle | grep name | grep -i $repo | head -n 1 | sed 's/name/rootProject.name =/g' | awk '{$1=$1};1'`
+  repo_name=$1
+  settings_gradle_content=`cat build.gradle | grep name | head -n 1 | grep -i ${repo_name:0:3} | sed 's/name/rootProject.name =/g' | awk '{$1=$1};1'`
```

Comment:
After talking to @zelinh this grep `grep -i ${repo_name:0:3}` is purely to catch the name var in perftop repo.
Since perftop repo is using the name `perf-top` instead of `perftop` repo name in build.gradle.

The better way to solve this is to add `settings.gradle` to repositories that lack this file, so whitesource script does not need to put in temp fixes to run the scan.